Alignment/OfflineValidation/plugins/PrimaryVertexValidation.h

```diff
@@ -1,77 +1,76 @@
 #ifndef PrimaryVertexValidation_h
 #define PrimaryVertexValidation_h
 
-#include "FWCore/Framework/interface/Frameworkfwd.h"
-#include "FWCore/Framework/interface/EDAnalyzer.h"
-#include "FWCore/Framework/interface/Event.h"
-#include "FWCore/Framework/interface/ESHandle.h"
-#include "FWCore/Framework/interface/EventSetup.h"
-#include "FWCore/Framework/interface/MakerMacros.h"
-#include "FWCore/ParameterSet/interface/ParameterSet.h"
+// system include files
+#include <string>
+#include <sstream>
+#include <vector>
+#include <map>
 
+// ROOT Included
 #include "TFile.h"
 #include "TH1D.h"
 #include "TH1I.h"
 #include "TH2D.h"
 #include "TTree.h"
 
-#include "DataFormats/TrackerRecHit2D/interface/SiStripRecHit2D.h"
-
-#include "DataFormats/TrajectorySeed/interface/TrajectorySeed.h"
-#include "DataFormats/TrajectorySeed/interface/TrajectorySeedCollection.h"
+// CMSSW includes
+#include "CommonTools/UtilAlgos/interface/TFileService.h"
+#include "DataFormats/Common/interface/DetSetVector.h"
+#include "DataFormats/DetId/interface/DetId.h"
+#include "DataFormats/GeometryVector/interface/GlobalPoint.h"
+#include "DataFormats/GeometryVector/interface/LocalPoint.h"
+#include "DataFormats/SiPixelDetId/interface/PixelSubdetector.h"
+#include "DataFormats/SiStripCluster/interface/SiStripCluster.h"
+#include "DataFormats/SiStripDetId/interface/StripSubdetector.h"
```

Comment:
was hoping not to need *Subdetector.h either - what triggers the need for these (getting to be obsolete..)